ositrace/ositrace_instance.json

```diff
@@ -63,7 +63,7 @@
           "@type": "xsd:string"
         }
       },
-      "ositrace:hasMovingObjects": {
+      "ositrace:hasTargetMovingObjects": {
```

Comment:
@jdsika Should this be plural or singular? I think for the most part, singular was used.